sshdconfig/src/args.rs

```diff
@@ -36,3 +39,9 @@ pub struct DefaultShell {
     pub escape_arguments: Option<bool>,
     pub shell_arguments: Option<Vec<String>>,
 }
+
+#[derive(Clone, Debug, Eq, PartialEq, ValueEnum)]
+pub enum Resource {
```

Comment:
Instead of using `resource` which is already overused, perhaps call this `Setting`?